index.js

```diff
@@ -23,5 +26,24 @@ module.exports.sync = (fp, data) => {
 
 	fp = path.basename(fp) === 'package.json' ? fp : path.join(fp, 'package.json');
 
+	data = normalize(data);
+
 	writeJsonFile.sync(fp, data, opts);
 };
+
+function normalize(pkg) {
+	const npkg = {};
+	for (const key of Object.keys(pkg)) {
+		switch (key) {
```

Comment:
Performance is relative. In this case, it would be a premature optimization, since you're not going to iterate over thousands or millions of properties. If we're talking micro-optimizations, you already lost lots of performance using a for-of loop, rather than a for-loop, which is a much bigger difference. But again, doesn't matter for this use-case :)